lib/node_modules/@stdlib/string/base/altcase/README.md

```diff
@@ -0,0 +1,102 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2022 The Stdlib Authors.
```

Comment:
@Jaysukh-409 You'll want to update the copyright year from 2022 to 2023 here and elsewhere in this PR.